data/assets/examples/celestial_globe.asset

```diff
@@ -0,0 +1,81 @@
+-- Model CC0 from https://sketchfab.com/3d-models/celestial-globe-341fa8a777e94883841409438756f747
+
+local sun = asset.require("scene/solarsystem/sun/transforms")
+local mars = asset.require("scene/solarsystem/planets/mars/mars")
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
+local moon = asset.require("scene/solarsystem/planets/earth/moon/moon")
+local lightsource = asset.require("scene/solarsystem/sun/light_source")
+
+local assetFolder = asset.resource({
+  Name = "Celestial Globe",
+  Type = "HttpSynchronization",
+  Identifier = "celestial_globe",
+  Version = 1
+})
+
+local globeModels = {}
+
+function createModel(longitude, latitude, scale, name, parent, group)
+  local model = {
+    Identifier = string.format("celestial-globe-%s", name),
+    Parent = parent,
+    Transform = {
+      Translation = {
+        Type = "GlobeTranslation",
+        Globe = parent,
+        Longitude = longitude,
+        Latitude = latitude,
+        UseHeightmap = true,
+        Altitude = 0.0
+      },
+      Rotation = {
+        Type = "GlobeRotation",
+        Globe = parent,
+        Longitude = longitude,
+        Latitude = latitude,
+        Angle= 180
+      },
+      Scale = {
+        Type = "StaticScale",
+        Scale = scale
+      }
+    },
+    Renderable = {
+      Type = "RenderableModel",
+      Enabled = true,
+      GeometryFile = assetFolder .. "celestial_globe.glb",
+      ModelScale = "Kilometer",
+      AmbientIntensity = 0.3,
```

Comment:
Since this is an example, do we want to explain these parameters in a comment? Or maybe keep it as simple as possible and not use these values? I'm trying to think what would be more understandable for a user who might read this and try to use it as a reference.